### PR TITLE
Remove default bcc

### PR DESCRIPTION
**Pull request by znarf**

### CODE REVIEW

File: server/lib/email.ts [hunk 1]
```diff
@@ -125,8 +125,6 @@ const sendMessage = (
   html: string,
   options: SendMessageOptions = {},
 ) => {
-  options.bcc = options.bcc || 'emailbcc@opencollective.com';
```
Comment (Betree) [inline server/lib/email.ts:128]: @znarf Should we keep it for staging? I still find it useful to have a place where I can get all staging emails.

Comment (znarf) [reply server/lib/email.ts:128]: I'm not relying on it personally. We don't send much emails on staging?

Why not look at Mailgun logs instead?

Comment (Betree) [reply server/lib/email.ts:128]: *Note: this was discussed in a call*

**Comment by Betree** [comment]
Related: I just noticed staging emails are sent from info@opencollective.com, which mixes them with the prod emails in mailgun's logs. @znarf should we change that?

**Review by Betree — APPROVED** [review]

**Pull request merged**